packages/arb-validator-core/message/message.go

```diff
@@ -119,6 +202,22 @@ func DeliveredValue(m Message) value.Value {
 	return msg
 }
 
+func AddToPrev(prev value.TupleValue, msg Message) value.TupleValue {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported function AddToPrev should have comment or be unexported](https://app.codacy.com/gh/OffchainLabs/arbitrum/pullRequest?prid=5428340)